src/main/xar-resources/data/urlrewrite/listings/listing-19.xml

```diff
@@ -0,0 +1 @@
+<clear-attribute name="xxx">
```

Comment:
@duncdrum FYI, this document that I added here wasn't well-formed, so I folded the fix into the rebased #906.